starport/cmd/scaffold_message.go

```diff
@@ -41,21 +44,31 @@ func messageHandler(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	var options []scaffolder.MessageOption
+
 	// Get description
 	desc, err := cmd.Flags().GetString(flagDescription)
 	if err != nil {
 		return err
 	}
-	if desc == "" {
-		// Use a default description
-		desc = fmt.Sprintf("Broadcast message %s", args[0])
+	if desc != "" {
+		options = append(options, scaffolder.WithDescription(desc))
+	}
+
+	// Get signer
+	signer, err := cmd.Flags().GetString(flagSigner)
```

Comment:
As we do for other commands lets define all flags in bulk in the beginning and we don't need to check the errors for them.